php-worker/Dockerfile

```diff
@@ -69,6 +69,12 @@ RUN if [ ${INSTALL_MYSQL_CLIENT} = true ]; then \
     apk --update add mysql-client \
 ;fi
 
+# Install FFMPEG:
+ARG INSTALL_FFMPEG=false
+RUN if [ ${INSTALL_FFMPEG} = true ]; then \
+    apk --update add mysql-client \
```

Comment:
why mysql-client ?

Reply:
Apologies, that was supposed to be `apk --update add ffmpeg`

Will update and create a new PR.
Great work you guys are doing though.